modules/ROOT/Edits/endpoint-ftp-send.adoc

```diff
@@ -0,0 +1,68 @@
+= FTP Send Endpoint Settings
+
+FTP send endpoint settings configure where to send messages in the following contexts:
+
+* Send to partners
++
+Target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to the partners
+
+* Target at host
++
+Target endpoint in an inbound message flow to which translated application JSON and XML messages are sent to the backend systems
+
+A host, partner, or 3rd party connection can own FTP send endpoints.
+
+Create FTP send endpoints by configuring these settings on the Host or Partner pages:
+
+[%header%autowidth.spread]
+|===
+|Name |Description | Required
```

Comment:
GLOBALLY standardize the style used for Description column texts: some begin with an article, and others with only the noun. Search throughout doc.